pmd-java/src/main/java/net/sourceforge/pmd/lang/java/oom/ClassStats.java

```diff
@@ -159,14 +159,14 @@
      *
      * @return The result of the computation, or {@code Double.NaN} if it couldn't be performed
      */
-    /* default */ double computeWithResultOption(OperationMetricKey key, ASTAnyTypeDeclaration node,
+    /* default */ double computeWithResultOption(MetricKey<OperationMetric> key, ASTAnyTypeDeclaration node,
```

Comment:
Hmm that's an interesting point. I began to do that since I saw [this comment](https://github.com/pmd/pmd/pull/444#discussion_r122558667) on a PR for type resolution. I thought it was PMD style, and I don't think it's too noisy so I kept it. 

The important thing is to have a consistent style though, so I guess that should be discussed in #361